### ISSUE TITLE
unable to login

### ISSUE THREAD

**Issue by huanz** [travis-ci/travis.rb#374]
run `travis login` got the following error:

```
System
Ruby:                     Ruby 2.3.0-p0
Operating System:         Mac OS X 10.11.3
RubyGems:                 RubyGems 2.6.1

CLI
Version:                  1.8.2
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       2016-03-02 13:08:20 +0800

Session
API Endpoint:             https://api.travis-ci.org/
Logged In:                no
Verify SSL:               yes
Enterprise:               no

Endpoints
org:                      https://api.travis-ci.org/ (current)

Last Exception
An error occurred running `travis login`:
    RuntimeError: Validation Failed
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:216:in `rescue in basic_auth'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:208:in `basic_auth'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:221:in `login'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:80:in `possible_tokens'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:49:in `each_token'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/tools/github.rb:36:in `with_token'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/login.rb:31:in `login'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/login.rb:40:in `run'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/command.rb:198:in `execute'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli.rb:64:in `run'
        from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:22:in `load'
        from /usr/local/bin/travis:22:in `<main>'


For issues with the command line tool, please visit https://github.com/travis-ci/travis.rb/issues.
For Travis CI in general, go to https://github.com/travis-ci/travis-ci/issues or email support@travis-ci.com.
```

**Comment by trinonsense** [comment]
i have similar issue, but different output

```
$ travis login
/usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/client/session.rb:221:in `raw': invalid access token (Travis::Client::NotLoggedIn)
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/client/session.rb:333:in `check_ssl'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/client/session.rb:41:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/client.rb:36:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/client.rb:36:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/api_command.rb:41:in `initialize'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/parser.rb:36:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli/parser.rb:36:in `new'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/lib/travis/cli.rb:62:in `run'
	from /usr/local/lib/ruby/gems/2.3.0/gems/travis-1.8.2/bin/travis:18:in `<top (required)>'
	from /usr/local/bin/travis:22:in `load'
	from /usr/local/bin/travis:22:in `<main>'
```

**Comment by septemus** [comment]
Bump. I get same issue as huanz.

**Comment by cmorent** [comment]
Same problem here

**Comment by ctriv** [comment]
Same here.  Seems that any command that goes over the network is failing.

**Comment by ray26** [comment]
Same here

**Comment by benediktvaldez** [comment]
Same here – @trinonsense did you ever figure out what the problem was?

**Comment by trinonsense** [comment]
hah nope. gave up.

**Comment by benediktvaldez** [comment]
@trinonsense just ran into #406 – worked for me

**Comment by carlad** [comment]
i've merged #406
